.devcontainer/post-create.sh

```
@@ -0,0 +1,2 @@
+# Set the default zsh theme user to vscode
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)